modular_zubbers/code/modules/storyteller/storyteller_vote.dm

```diff
@@ -23,17 +21,11 @@
 
 /datum/vote/storyteller/initiate_vote(initiator, duration)
 	. = ..()
-	to_chat(world, vote_font(fieldset_block("Storyteller Vote", "[span_vote_notice("Only players who are ready and joining the game round start will be calculated in voting results.")]", "boxed_message purple_box")))
+	to_chat(world, vote_font(fieldset_block("Storyteller Vote", "[span_vote_notice("All votes matter!")]", "boxed_message purple_box")))
```

Comment:
```suggestion
	to_chat(world, vote_font(fieldset_block("Storyteller Vote", "[span_vote_notice("All votes matter!")]", "boxed_message purple_box")))
```
Just remove this entirely. We don't need two message boxes for a vote initiation.